conf/st2.package.conf

```diff
@@ -52,7 +52,7 @@ api_url =
 base_path = /opt/stackstorm
 
 [syslog]
-host = localhost
+host = 127.0.0.1
```

Comment:
We can not be sure about other services. Ours we explicitly bind to this address.